src/inference/dev_api/cpp_interfaces/impl/ie_infer_async_request_thread_safe_default.hpp

```diff
@@ -16,9 +16,11 @@
 
 #include "cpp_interfaces/interface/ie_iinfer_request_internal.hpp"
 #include "ie_api.h"
-#include "threading/ie_immediate_executor.hpp"
-#include "threading/ie_istreams_executor.hpp"
-#include "threading/ie_itask_executor.hpp"
+#include "openvino/runtime/threading/immediate_executor.hpp"
+#include "openvino/runtime/threading/istreams_executor.hpp"
+#include "openvino/runtime/threading/itask_executor.hpp"
+
+using namespace ov::threading;
```

Comment:
This file is legacy, and will be removed in 24.0, not need to change it.

Reply:
ok, updated.